build/NuSpecs/ProjectReunion-Nuget-Native.WinRt.props

```diff
@@ -6,20 +6,20 @@
     <_ProjectReunionFoundationPlatform Condition="'$(Platform)' != 'Win32'">$(Platform)</_ProjectReunionFoundationPlatform>
   </PropertyGroup>
 
-  <!-- Some of the manual reference adding below may not be necessary due to the files in question being in default locations, but there's no harm in it. -->
-
-  <!-- OutputType Exe corresponds to C++ -->
-  <ItemGroup Condition="'$(OutputType)' == 'Exe'">
-    <Reference Include="Microsoft.Windows.AppLifecycle.winmd">
-      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.Windows.AppLifecycle.winmd</HintPath>
-      <Implementation Condition="'$(ProjectReunionFrameworkPackage)' != 'true'">$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_ProjectReunionFoundationPlatform)\native\Microsoft.Windows.AppLifecycle.dll</Implementation>
-      <IsWinMDFile>true</IsWinMDFile>
-    </Reference>
+  <ItemGroup Condition="'$(AppxPackage)' != 'true'">
     <Reference Include="Microsoft.ApplicationModel.DynamicDependency.winmd">
-      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\uap10.0\Microsoft.ApplicationModel.DynamicDependency.winmd</HintPath>
+      <HintPath>$(MSBuildThisFileDirectory)..\..\lib\native\Microsoft.ApplicationModel.DynamicDependency.winmd</HintPath>
       <Implementation Condition="'$(ProjectReunionFrameworkPackage)' != 'true'">$(MSBuildThisFileDirectory)..\..\runtimes\win10-$(_ProjectReunionFoundationPlatform)\native\Microsoft.ApplicationModel.DynamicDependency.dll</Implementation>
       <IsWinMDFile>true</IsWinMDFile>
     </Reference>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
In general, we need to probably take a look at grooming this to be a LOT easier.  Adding things to the builds shouldn't require our developers to make this type of extensive changes.  Obviously it's just because of the infancy of the project, but I would still like to just call it out.

Reply:
This was based on previous work (MRT) but yeah, I'm unsure how much of that was necessary or just what-seemed-like-a-good-idea-at-the-time. We'll want to take an overall deepthink across all the Nuspecs\* tree. If we're going to dynamically generate content for appxmanifest.xml we should be dynamically generating a bunch of content here too.

Something for a later day